Arm/BitVec.lean

```diff
@@ -294,7 +294,7 @@ abbrev partInstall (hi lo : Nat) (val : BitVec (hi - lo + 1)) (x : BitVec n): Bi
   let x_with_hole := x &&& mask_with_hole
   x_with_hole ||| val_aligned
 
-example : (partInstall 3 0 0xC#4 0xAB0D#16 = 0xAB0C#16) := by native_decide
+example : (partInstall 3 0 0xC#4 0xAB0D#16 = 0xAB0C#16) := by rfl
```

Comment:
```suggestion
example : (partInstall 3 0 0xC#4 0xAB0D#16 = 0xAB0C#16) := rfl
```

(minor nit) If a proof is just `rfl`, we prefer term-mode over tactic mode

Reply:
I've fixed a bunch of these in the repo, see new commit.